src/Server/Config/router-config.php

```diff
@@ -163,6 +163,28 @@
                     ['countRequests' => null],
                 ],
                 'else' => [
+                    ['goto' => 'verifyMethodIsPut'],
+                ],
+            ],
+        ],
+        'verifyMethodIsPut' => [
+            'condition' => [
+                'all-of' => [
+                    [
+                        'input-source' => ['method' => null],
+                        'matcher'      => ['isEqualTo' => 'PUT'],
```

Comment:
I've used PUT method for the callback, it isn't the best solution, but requests are processed by this (executions) section, and POST is already used. I didn't extended the URL as I see URLs are simple for all callbacks. 

The options could be to use POST for this, but extend URLs for both added and count callbacks like `executions/list` and `executions/count`, or separate them with other params.

Reply:
Hey. I think having __phiremock/executions as a url could be a good idea. It represents what we are requesting and also we can use GET as verb, that also represents better the retrieval of the executions. What do you think?
If you agree I can do the change myself after merging.

Reply:
Sorry, I didn't caught the idea. I've used `PUT __phiremock/executions` for the action. Do you propose use `GET __phiremock/executions` for this?
If it so, I don't think that GET method is better - see https://stackoverflow.com/a/983458.

Reply:
I see. So it was a very good idea to have PUT as verb.